src/Math-Matrix/PMTensor.class.st

```diff
@@ -0,0 +1,145 @@
+Class {
+	#name : #PMTensor,
+	#superclass : #Object,
+	#instVars : [
+		'array',
+		'shape',
+		'first',
+		'strides'
+	],
+	#category : #'Math-Matrix'
+}
+
+{ #category : #'instance creation' }
+PMTensor class >> newWith: anInteger [
```

Comment:
I would appreciate this being a little more intention revealing: is this the dimension of the tensor? The rank?

Reply:
This is a method to construct rank 0 (scalar) vector.